requirements.txt

```diff
@@ -30,7 +30,7 @@ v3io-frames~=0.10.12
 semver~=3.0
 dependency-injector~=4.41
 # should be identical to gcs and s3fs.
-fsspec==2023.9.2
+fsspec>=2023.9.2, <2024.4
```

Comment:
does it actually select the newer fsspec when adlfs, ossfs are behind? Or they support the newer fsspec?

Reply:
* adlfs 2023.9.0 [requires](https://github.com/fsspec/adlfs/blob/2023.9.0/setup.py#L36) `fsspec>=2023.9.0`
* ossfs 2023.12.0 [requires](https://github.com/fsspec/ossfs/blob/2023.12.0/setup.cfg#L29) `fsspec>=2023.9.2`

So in both cases, there is no upper bound that would restrict fsspec.